### ISSUE TITLE
[v1.9.10] 'ws-send' sends message twice when triggered with 'load'

### ISSUE THREAD

**Issue by zarekr** [bigskysoftware/htmx#2141]
With ver 1.9.10 I have noticed that, when using ws-send on an element to send to an open websocket connection, using hx-trigger="load", the message is sent twice. For example: 

```
<div ws-connect="/ws/some/url">

<div id="some_id" ws-send hx-trigger="load" hx-vals='{"some_key":"some_val"}'></div>

</div>
```

In the network pane of browser dev tools, I'll see this twice as a sent message when using v1.9.10 and only once with 1.9.9:

```
{"some_key": "some_val", "HEADERS":{"HX-Request":"true","HX-Trigger":"some_id", "HX-Trigger-Name":null,"HX-Target":"some_id","HX-Current-URL":"http://127.0.0.1:8000/home"}}	1703969407.3404334

{"some_key": "some_val", "HEADERS":{"HX-Request":"true","HX-Trigger":"some_id", "HX-Trigger-Name":null,"HX-Target":"some_id","HX-Current-URL":"http://127.0.0.1:8000/home"},	1703969407.3404334
```

As a result the backend route handling the ws message processes it twice. 

I'm guessing it is to do with load being fired an additional time but I can't find where this is occurring. Looking at the changes it may be to do with the new readystate stuff.

**Comment by DaveManders2205** [comment]
This problem also occurred to me.
This is the temporary workaround im using until it gets fixed: `hx-trigger="load delay:1ms"`

**Comment by johannesschaffer** [comment]
Doesn't work for me with v1.9.9 or below either. First time I'm using HTMX, so maybe I'm doing something wrong?
```
<form hx-ws='send' hx-ext="ws" ws-connect="/chat" class="flex items-center gap-2">
        <input placeholder="Type a message" name='message'
```

**Comment by johannesschaffer** [comment]
For me the problem was I used the old attribute: `hx-ws='send'` instead of `hx-boost="" ws-send=""`

**Comment by CaliViking** [comment]
I have the same issue.
I have also noticed that the units of delay do not seem to work. Both ```delay:1s``` and ```delay:1ms``` seem to delay by 1 second. When I try to use ```delay:1h```, then the response is immediate.

Even stranger, when I use ```delay:1h``` the result is immediate and not repeated, but when I use ```delay:0s``` the response is immediate and duplicated.

I can't find this in the documentation, but ```delay:0.9s``` takes longer than ```delay:0.1s```, so there must be support for real numbers. I also tried ```delay:0.001s``` which seems to work, but it is hard to measure with the naked eye.

**Comment by thiagomagro** [comment]
Same behavior here. Load fires ws-send twice.

**Comment by 1cg** [comment]
no longer relevant
